bitseq/sequence.go

```diff
@@ -498,24 +506,40 @@ func (h *Handle) UnmarshalJSON(data []byte) error {
 func getFirstAvailable(head *sequence, start uint64) (uint64, uint64, error) {
 	// Find sequence which contains the start bit
 	byteStart, bitStart := ordinalToPos(start)
```

Comment:
as a reminder for myself, byteStart and bitStart here are relative to the first byte of the block where the ordinal start falls into